src/edutap/wallet_google/modelcore.py

```diff
@@ -12,7 +12,7 @@ class GoogleWalletModel(BaseModel):
     """
 
     model_config = ConfigDict(
-        # extra="forbid",
+        extra="forbid",
         # extra="ignore",
         # use_enum_values=True,
```

Comment:
I am pretty sure we want to use_enum_values to be true on serialization
- https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.use_enum_values
- https://stackoverflow.com/a/76973367/227245